api/src/main/java/org/pmiops/workbench/billing/BillingProjectBufferService.java

```diff
@@ -59,6 +78,20 @@ public BillingProjectBufferService(
     this.clock = clock;
     this.fireCloudService = fireCloudService;
     this.workbenchConfigProvider = workbenchConfigProvider;
+    View view = View.create(
+        View.Name.create("billing_project_buffer_entries"),
+        "The number of billing project buffer entries.",
+        BUFFER_ENTRY_MEASUREMENT,
+        Aggregation.Distribution.create(PROJECT_BOUNDARIES),
+        Collections.emptyList());
+    ViewManager viewManager = Stats.getViewManager();
+    viewManager.registerView(view);
+    try {
+      StackdriverStatsExporter.createAndRegister();
+    } catch (IOException e) {
+      log.log(Level.WARNING, "Failure to initialize stackdriver stats exporter");
```

Comment:
This is going to need to be ERROR or SEVERE so that our log-based alerts will let us know the primary alert channel is down.